source/_integrations/zwave_js.markdown

```diff
@@ -181,7 +183,7 @@ Do this if you have an existing Z-Wave network and want to use a new controller.
 ### To migrate a Z-Wave network to a new controller
 
 1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & services**" %}.
-2. Select the **Z-Wave** integration. Then, select **Configure**.
+2. Select the **Z-Wave** integration. Then, select the cogwheel {% icon "mdi:cog-outline" %}.
```

Comment:
Should we use a bullet for "Then, select the cogwheel" like the other instructions?